### ISSUE TITLE
UnicodeDecodeError: 'cp950' codec can't decode byte 0xbf in position 2: illegal multibyte sequence

### ISSUE THREAD

**Issue by aappaapp** [XroixHD/MCBE-Win10-FOV-Changer#128]
```
----------------------------------------[27.05.2021 - 18:14:07]----------------------------------------
[18:13:50] + Root Thread
[18:13:50] + ProcessingThread
[18:13:50] + System Tray
[18:13:50] + Storage
[18:13:50] + Network
[18:13:50] + Gateway
[18:13:50] + Discord
[18:13:50] + Root Gui
[18:13:51] > Rendered Settings!

[18:14:07] > Attached to Minecraft!
Traceback (most recent call last):
  File "src\thread.py", line 82, in run
  File "src\processing\processing.py", line 177, in start_button_handle
  File "src\processing\processing.py", line 565, in check_version
  File "src\processing\processing.py", line 553, in get_mc_version
UnicodeDecodeError: 'cp950' codec can't decode byte 0xbf in position 2: illegal multibyte sequence


```

**Comment by XroixHD** [comment]
Should be fixed in the beta (http://www.mediafire.com/file/ay5z53t9logj2wz/FOV-Changer-1.1.6-beta.zip/file)
And I know that this version is in the beta state for almost a year now. I hope I will get time soon to work on this project more frequently.

**Comment by xroix** [comment]
Info: Has been fixed in 6d75b0275ce3121fe3c0607281fd6cf147627f8d and a1f2af09227566c3568f4fdcd1735d05aa1616af
